index.html

```diff
@@ -0,0 +1,184 @@
+                </div>
+                <div class="col-md-4 mt-4">
+                    <div class="card">
+                        <i class="fas fa-comments fa-4x"></i>
+                        <h3>QA</h3>
+                        <h6>12:00 p.m - 12:30 p.m </h6>
+                    </div>
+                </div>
+            </div>
+            <div class="row mt-4">
+                <div class="col-md-4 mt-4">
+                    <div class="card">
+                        <i class="fas fa-tools fa-4x"></i>
+                        <h3>Workshop</h3>
+                        <h6>2:00 p.m - 4:00 p.m</h6>
+                    </div>
+                </div>
+                <div class="col-md-4 mt-4">
+                    <div class="card">
+                        <i class="fas fa-fire fa-4x"></i>
+                        <h3>Team Exercise</h3>
+                        <h6>4:00 p.m - 6:00 p.m</h6>
+                    </div>
+                </div>
+            </div>
+            <div class="row mt-4">
+                <div class="col-md-12 text-center">
+                    <button id="showProgramButton" class="btn btn-primary">See the Whole Program</button>
+                </div>
+            </div>
+            <div id="programCards" class="row mt-4">
+                <div class="col-md-4 mt-4">
+                    <div class="program-card card">
+                        <i class="fas fa-chalkboard-teacher fa-4x"></i>
+                        <h3>Additional Lecture</h3>
+                        <h5>Announcement soon</h5>
+                    </div>
+                </div>
+                <div class="col-md-4 mt-4">
+                    <div class="program-card card">
+                        <i class="fas fa-museum fa-4x"></i>
+                        <h3>Additional Exhibition</h3>
+                        <h6>Announcement soon</h6>
+                    </div>
+                </div>
+                <div class="col-md-4 mt-4">
+                    <div class="program-card card">
+                        <i class="fas fa-comments fa-4x"></i>
+                        <h3>Additional Forum</h3>
+                        <h6>Announcement soon</h6>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section id="speakers" class="about">
+        <h2>Featured Speakers</h2>
+        <p>Our API Festival brings together some of the brightest minds in the industry. Learn from industry experts,
+            thought leaders, and innovators who have pioneered the world of APIs.</p>
+        <p>Below is the list of our esteemed lineup of speakers.</p>
+        <div class="container mt-4">
+            <h2>Speakers List</h2>
+            <div class="row" id="speakerContainer">
+            </div>
+            <div id="seeMoreButton">
+                <button class="btn btn-primary" onclick="showHiddenSpeakers()">See More</button>
+            </div>
+        </div>
+    </section>
```

Comment:
- [ ] The "Featured speakers" section should also follow the Behance design so please make sure you have 2 columns of 3 speakers each in the desktop version and make sure the design is a visual copy of the Behance design.
- Behance design:
  ![image](https://github.com/NgachaIan/Ngacha_Portfolio/assets/104968236/420567a7-b6f3-4e96-86f8-f2f179938eb0)
- Your design:
  ![image](https://github.com/NgachaIan/Ngacha_Portfolio/assets/104968236/7e9a1c51-bd54-48d3-9e96-6c25c8af55cb)
- [ ] This also applies to the mobile version so please revisit the [Benahce design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and structure your design to match it.
- Behance design:
  ![image](https://github.com/NgachaIan/Ngacha_Portfolio/assets/104968236/9dac40e2-e01a-48b4-a332-2579dccfedf6)
- Your design:
  ![image](https://github.com/NgachaIan/Ngacha_Portfolio/assets/104968236/c3eb05bc-5911-4ecf-830a-500a7a0d3102)